jobserver/urls.py

```diff
@@ -148,7 +153,8 @@
         name="project-membership-remove",
     ),
     path("new-workspace/", WorkspaceCreate.as_view(), name="workspace-create"),
-    path("releases/", Releases.as_view(), name="project-releases"),
+    path("releases/", ProjectReleaseList.as_view(), name="project-release-list"),
+    path("releases/spa/", Releases.as_view(), name="project-releases"),
```

Comment:
Hmm, not sure I like `spa` in the url. What about "releases/view"?

Reply:
This is just a placeholder to move the SPA somewhere else, #655 will move it to a more relevant place.